client/packages/common/src/utils/numbers/NumUtils.ts

```diff
@@ -12,4 +17,19 @@ export const NumUtils = {
 
     return constrain(parsed, min, max);
   },
+  /**
+   * This constant should be used for values that are potentially send to a backend API that expects
+   * an unsigned 32 bit integer and thus would reject Number.MAX_SAFE_INTEGER.
+   * For example, JS number max size is `2^53 - 1` while the Rust u32 size is `2^32 - 1`.
+   */
+  MAX_SAFE_API_INTEGER: 999999999,
```

Comment:
Where does this `999999999` come from? Why not `4294967295` (`2^32 - 1`)?

Reply:
This was the previous value... @mark-prins was this an arbitrary choice?

Reply:
yes, iirc just an arbitrary big number. when we created the controls we were dealing with pack sizes and quantities, and reasoned that it is unlikely anyone will order `2^32 - 1` packs 😉